src/webapp/pages/app/App.tsx

```diff
@@ -38,7 +39,7 @@ function App(props: AppProps) {
             const orgUnits = await compositionRoot.orgUnits.getAll.execute().toPromise();
 
             const configurations = await compositionRoot.diseaseOutbreakEvent.getConfigurations
-                .execute()
+                .execute({ incidentManagerUserGroupCode: RTSL_ZEBRA_INCIDENTMANAGER })
```

Comment:
do we need to pass incidentManagerUserGroupCode from here ? It is defined in the repository, so it can be used directly in the getUserGroupByCode. We do not need to send it from presentation to domain to data layer again. It belongs in the data layer and can shared directly between the 2 repos.

Reply:
okay understood 👍